1.7/scala_2.12-debian/docker-entrypoint.sh

```diff
@@ -56,7 +59,7 @@ elif [ "$1" = "taskmanager" ]; then
 
     echo "Starting Task Manager"
     echo "config file: " && grep '^[^\n#]' "$FLINK_HOME/conf/flink-conf.yaml"
-    exec $(drop_privs_cmd) "$FLINK_HOME/bin/taskmanager.sh" start-foreground
+    exec $(drop_privs_cmd) "$FLINK_HOME/bin/taskmanager.sh" start-foreground "${ARGS[@]}"
```

Comment:
It looks like that other images' entry points are also missing "varargs" ([hadoop-version-]scala_[2.11|2.12]-[alpine|debian]).

I think you can pass them as `"$@"` like [here](https://github.com/docker-flink/docker-flink/blob/master/1.7/scala_2.12-debian/docker-entrypoint.sh#L48).

You wouldn't need to change `#!/bin/sh`.

Reply:
Yes, in images' entry points are also missing "varargs".

If you add `$@`, then the `taskmanager` argument will be passed to taskmanager.sh, which is passed to docker-entrypoint.sh earlier.

If we use `$@`, then in taskmanager.sh/jobmanager.sh we have to replace the [next line](https://github.com/apache/flink/blob/master/flink-dist/src/main/flink-bin/bin/taskmanager.sh#L25) with `ARGS=("${@:3}")`

Reply:
The `"jobmanager"` branch [calls `shift 1`](https://github.com/docker-flink/docker-flink/blob/master/1.7/scala_2.12-debian/docker-entrypoint.sh#L41) to drop the first argument from the `$@`. The same can be done for `"taskmanager"` part and it would make both branches more consistent.

Reply:
Yes, thank you, did not see this line. Added `shift 1` for `taskmanager` part.

Reply:
@tarmazakov, FYI, it looks like the `docker-entrypoint.sh` in all those directories are not manually edited, but rather they are just copies of [the file in the root of the project](https://github.com/docker-flink/docker-flink/blob/master/docker-entrypoint.sh). You may need to update only that copy.

The project maintainers can correct me if I'm wrong.

Reply:
Of course, manual editing of all files is not the best idea. Let's wait for the response of the project maintainers.